dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7728,6 +7728,21 @@ public String toString() {
           .setScope(Scope.WORKER)
           .build();
 
+  public static final PropertyKey WORKER_HTTP_SERVER_ENABLED =
+      booleanBuilder(Name.WORKER_HTTP_SERVER_ENABLED)
+          .setDefaultValue(true)
+          .setDescription("Whether to enable worker HTTP server.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
+          .setScope(Scope.WORKER)
+          .build();
+
+  public static final PropertyKey WORKER_HTTP_SERVER_PORT =
+      intBuilder(Name.WORKER_HTTP_SERVER_PORT)
+          .setDefaultValue(28080)
```

Comment:
Is this port following some rule? Or just pick a random one?

Reply:
Just pick a random one.